### PR TITLE
Added call notes for ACD #107

### PR DESCRIPTION
**Pull request by shanelightowler**

### CODE REVIEW

**Comment by shanelightowler** [comment]
Apologies, this PR contains both ACD #107 notes as well as the EIP-1559 community call transcript/notes. They are both my final versions for review but am happy to re-upload as seperate PRs if required.

**Comment by poojaranjan** [comment]
One PR should be fine, just make sure they are going into the correct folders. Could you also update the README section for both notes?
Thanks!

File: All Core Devs Meetings/Meeting 107.md [hunk 1]
```diff
@@ -0,0 +1,514 @@
...
+
+* Agrees with Tim re: 3198, will re-write reasoning for this.
+
+**Hudson**
+
+* Let's discuss 3198 in ACD Discord and next ACD call.
+
+**Peter**
+
+* Alexi should be brought into that discussion.
```
Comment (poojaranjan) [inline All Core Devs Meetings/Meeting 107.md:467]:
```suggestion
* Alexey should be brought into that discussion.
```

File: All Core Devs Meetings/Meeting 107.md [hunk 2]
```diff
@@ -0,0 +1,514 @@
...
+* Strongest motivation for removing refunds is limited capacity. We should be well below that... 1/4 capacity.
+* Sceptical that 10x would stop the network.
+* Supports increasing the gas limit and 1559/gas target.
+* In favour of EIP-3322 in this area.
+
+**Hudson**
+
+* No time for rebuttals on this - post rebuttals in the [agenda topic](https://github.com/ethereum/pm/issues/258) for EIP-3300.
+* William is on Eth R&D Discord for questions/rebuttals.
+* Goodbye all, thanks for all the fish.
```
Comment (poojaranjan) [inline All Core Devs Meetings/Meeting 107.md:484]:
```suggestion
* Goodbye all, thanks for having me do this last few years. Let's all get excited for Tim taking this over. He is going to do a great job. Thank you all, we've another meeting in two weeks on March 19th at 1400 UTC.```

**Review by poojaranjan — COMMENTED** [review]
Looks good, minor edits requested. Thanks!

**Comment by shanelightowler** [comment]
@poojaranjan I have committed your changes. I wonder if the readmes should be left to @timbeiko for these? I know the structure of the agenda/notes layout on the readme for ACD (and presume 1559) are in discussion as to best format at the moment?

**Comment by timbeiko** [comment]
@shanelightowler I'll merge this for now and do the README shortly. Thank you!

**Pull request merged**